src/bp/ui-studio/src/web/views/OneFlow/sidePanel/WorkflowEditor/import.ts

```diff
@@ -99,8 +99,9 @@ export const executeWorkflowActions = async (actions: ImportAction[]) => {
   }
 
   try {
-    await Promise.each(getActionsForType('intent'), ({ data }) =>
-      axios.post(`${window.BOT_API_PATH}/nlu/intents`, data)
+    await Promise.each(
+      getActionsForType('intent'),
+      ({ data }) => axios.post(`${window.BOT_API_PATH}/nlu/intents`, data) // TODO: remove this (can't import intents anymore, they are part of files)
```

Comment:
needs to parse current flow to get intents. 

so we need  `load intents from flow` for ui purposes as well